core/domain/param_domain_test.py

```diff
@@ -28,11 +28,7 @@ class ParameterDomainUnitTests(test_utils.GenericTestBase):
 
     def test_param_spec_validation(self):
         """Test validation of param specs."""
-        param_spec = param_domain.ParamSpec('FakeType')
-        with self.assertRaisesRegex(TypeError, 'is not a valid object class'):
-            param_spec.validate()
-
-        param_spec.obj_type = 'Real'
```

Comment:
I think this test should still be kept (or at least moved).

Reply:
I think it depends on the below conversation. if we can move that then surely we can check this as well.